src/Config.cpp

```diff
@@ -59,6 +67,14 @@ ConfigEntry ConfigFile[] =
     {"DLDIEnable", 0, &DLDIEnable, 0, NULL, 0},
     {"DLDISDPath", 1, DLDISDPath, 0, "", 1023},
 
+    {"FirmwareUsername", 1, FirmwareUsername, 0, "MelonDS", 63},
```

Comment:
melonDS is the proper capitalisation

Reply:
Might be a good time to ask what the default username *should* be?

Reply:
if it's going to be melonDS it should atleast be capitalized correctly :P 

other than that, I'm currently out of ideas, other than stupid shit like 'Kagamine Len'

maybe we should run a poll somewhere?